src/Tempest/Mailer/src/Email.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Mailer;
+
+use Tempest\Mailer\Components\Address\ImmutableAddressCollection;
+
+final readonly class Email
+{
+    public function __construct(
```

Comment:
Shouldn't an email have a body?

Reply:
As well as optional attachements?

Reply:
Ah it's probably just WIP :)

Reply:
Just POC to ensure we agree on the format and then I'll fill it in.